src/lean_spec/types/ssz_base.py

```diff
@@ -102,43 +100,49 @@ class SSZModel(StrictBaseModel, SSZType):
     """
     Base class for SSZ types that use Pydantic validation.
 
-    This combines StrictBaseModel (Pydantic validation + immutability) with SSZ serialization.
-    Use this for containers and complex types that can benefit from Pydantic.
+    This combines StrictBaseModel (Pydantic validation + immutability)
+    with SSZ serialization. Use this for containers and complex types that
+    can benefit from Pydantic.
 
-    For simple types that need special inheritance (like int), use SSZType directly.
+    For simple types that need special inheritance (like int), use SSZType
+    directly.
 
-    SSZModel provides natural iteration and indexing for collections with a 'data' field:
+    SSZModel provides natural iteration and indexing for collections with a
+    'data' field:
```

Comment:
These should also go in a separate PR. Also seems like a different linter config